.jenkins

```diff
@@ -314,8 +314,67 @@ pipeline {
                         }
                     }
                 }
+
+                stage('HIP-3.5') {
+                    agent {
+                        dockerfile {
+                            filename "Dockerfile.hipcc"
+                            dir "docker"
+                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-18.04:3.5'
+                            args '-v /tmp/ccache.kokkos:/tmp/ccache --device=/dev/kfd --device=/dev/dri --security-opt seccomp=unconfined --group-add video'
+                            label 'rocm-docker && vega'
+                        }
+                    }
+                    steps {
+                        sh 'ccache --zero-stats'
+                        sh 'rm -rf build && mkdir -p build'
+                        dir('build') {
+                            sh '''
+                                cmake \
+                                    -D CMAKE_INSTALL_PREFIX=$ARBORX_DIR \
+                                    -D CMAKE_BUILD_TYPE=Release \
```

Comment:
Did you have a chance to try Debug on Tulip?

Reply:
Yes, same problem.